src/main/java/com/mapzen/pelias/Pelias.java

```diff
@@ -45,31 +46,41 @@ public static Pelias getPelias() {
     }
 
     public void suggest(String query, String lat, String lon, Callback<Result> callback) {
```

Comment:
I think there is an opportunity to cleanup these overloaded methods by making the version with fewer arguments call the version with more arguments. For example:

```java
    public void suggest(String query, Callback<Result> callback) {
        suggest(query, locationProvider.getLat(), locationProvider.getLon(), callback);
    }
    
    public void suggest(String query, String lat, String lon, Callback<Result> callback) {
        service.getSuggest(query, lat, lon, apiKey,  callback);
    }

    public void search(String query, String minLon, String minLat, String maxLon, String maxLat,
            Callback<Result> callback) {
        search(query, locationProvider.getLat(), locationProvider.getLon(),
                minLon, minLat, maxLon, maxLat, callback);
    }

    public void search(String query, String lat, String lon, String minLon, String minLat,
            String maxLon, String maxLat, Callback<Result> callback) {
        service.getSearch(query, lat, lon, minLon, minLat, maxLon, maxLat, apiKey, callback);
    }
```

Also we may want to introduce a `BoundingBox` object to cut down on the number of parameters required by search.

```java
    public class BoundingBox {
        public String minLat;
        public String minLon;
        public String maxLat;
        public String maxLon;

        public BoundingBox(double minLat, double minLon, double maxLat, double maxLon) {
            this.minLat = Double.toString(minLat);
            this.minLon = Double.toString(minLon);
            this.maxLat = Double.toString(maxLat);
            this.maxLon = Double.toString(maxLon);
        }
    }
```

Then the calls to search get even cleaner :)

```java
    public void search(String query, BoundingBox box, Callback<Result> callback) {
        search(query, locationProvider.getLat(), locationProvider.getLon(), box, callback);
    }

    public void search(String query, String lat, String lon, BoundingBox box, Callback<Result> callback) {
        service.getSearch(query, lat, lon, box.minLon, box.minLat, box.maxLon, box.maxLat, apiKey, callback);
    }
```